CodeDrivers/CodeDrivers/CodeDrivers/CsvHanlder.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeDrivers
+{
+    internal class CsvHanlder
+    {
+        private string filePath = @"C:\Users\Janek\Desktop\FakeCrudentials.csv";
+        public List<string> GetCredentialsFromFile(string path)
+        {
+            var credentials = new List<string>();
+            try
+            {
+                string[] lines = File.ReadAllLines(filePath);
+                credentials.AddRange(lines);
+
+                return credentials;
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Wystąpił błąd podczas odczytu pliku CSV: " + ex.Message);
+                return null;
+            }
+
+        }
+
+        public bool SelectRequestedValues(string login, string password, List<string> credentials)
```

Comment:
Ta metoda ma bardzo ogólną nazwę. Nic nie mówi o domenie w jakiej działa, w zasadzie po nazwie mogłaby być generyczna :) lepiej dobrać bardziej dookreśloną nazwę, np AuthorizeUser (bo w ciele metody widzę, że to właśnie robi)